api/migrations/versions/af9c317d2c92_.py

```diff
@@ -0,0 +1,522 @@
+    order = db.Column(db.Integer(), nullable=False)
+    depends_on_question_id = db.Column(db.Integer(), db.ForeignKey('question.id', use_alter=True), nullable=True)
+    show_for_values = db.Column(db.JSON(), nullable=True)
+
+    def __init__(self, application_form_id, name, description, order):
+        self.application_form_id = application_form_id
+        self.name = name
+        self.description = description
+        self.order = order
+
+def get_country_list(session):
+    countries = session.query(Country).all()
+    country_list = []
+    for country in countries:
+        country_list.append({
+            'label': country.name,
+            'value': country.name
+        })
+    return country_list
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    # pass
+    # ### end Alembic commands ###
+
+    Base.metadata.bind = op.get_bind()
+    session = orm.Session(bind=Base.metadata.bind)
+
+    maathaiimpact2020 = Event('Wangari Maathai Impact Award 2020',
+        'Wangari Maathai Impact Award 2020', 
+        datetime.date(2020, 8, 23), datetime.date(2020, 8, 28), 'maathai2020',
+        1, 'baobab@deeplearningindaba.com', 'http://www.deeplearningindaba.com', 
+        datetime.date(2020,3,1), datetime.date(2020,4,17), datetime.date(2020,4,25), 
+        datetime.date(2020,5,15),datetime.date(2020,1,1), datetime.date(2020,1,1), 
+        datetime.date(2020,1,1),datetime.date(2020,1,1), datetime.date(2020,1,1), 
+        datetime.date(2020,1,1), EventType.AWARD)
+    session.add(maathaiimpact2020)
+    session.commit()
+
+    event_id = maathaiimpact2020.id
+
+    application_form = ApplicationForm(event_id, True, True)
+    session.add(application_form)
+    session.commit()
+
+    app_form_id = application_form.id
+
+    main_section = Section(app_form_id, 'Wangari Maathai Impact Award 2020', """
+This is the official application form for the Wangari Maathai Impact Award 2020, an award to encourage and recognise work by African innovators that shows impactful application of machine learning and artificial intelligence. This award will be made at the Deep Learning Indaba in Tunis, Tunisia in August 2020.
+
+This application will require:
+- Personal details about the nominee,
+- Details about the impactful work, including why it is impactful, who it impacts and why is it innovative,
+- Details of 2 people other than the nominator to provide supporting letters for the nominee
+
+For eligibility criteria for the Maathai Award, please see www.deeplearningindaba.com/maathai-2020
+
+For any queries, please email awards@deeplearningindaba.com.
+    """, 1)
+    session.add(main_section)
+    session.commit()
+
+    q1_nomination_capacity = Question(
+        application_form_id=app_form_id,
+        section_id=main_section.id,
+        headline='Nominating Capacity',
+        placeholder='',
+        order=1,
+        questionType='multi-choice',
+        validation_regex=1,
```

Comment:
The validation_regex should be `None` when no validation is required. (Same comment applies to questions below)